math/genvector/inc/Math/GenVector/VectorUtil.h

```diff
@@ -39,7 +39,7 @@ namespace ROOT {
 
        @ingroup GenVector
 
-       @sa Overview of the @ref GenVector "physics vector library"
+       @sa GenVector "physics vector library"
```

Comment:
```suggestion
       @note Overview of the @ref GenVector "physics vector library"
```

The \sa command is only guaranteed to work well if all what is inside are classes or function names / files separated by commas. Otherwise, it might or might not render correctly the output.

Reply:
I'm not sure that's correct. Please check the current docs with two broken links: "Overview" and "the" go to `TRangeDynCast`.
![image](https://github.com/user-attachments/assets/8e67fa3a-ecac-4b1d-aba5-0717f73ba1db)

Compare that to something that I generated *locally* from this PR:
![image](https://github.com/user-attachments/assets/7fade39d-bffd-4ba6-9ee5-f8fcdac9e709)

Unless I'm using a different version of Doxygen, the changes proposed in the PR are better.